numpy/ma/core.py

```diff
@@ -191,14 +191,17 @@ class MaskError(MAError):
     default_filler["M8[" + v + "]"] = np.datetime64("NaT", v)
     default_filler["m8[" + v + "]"] = np.timedelta64("NaT", v)
 
+float_types_list = [np.half, np.single, np.double, np.longdouble,
+                    np.csingle, np.cdouble, np.clongdouble]
 max_filler = ntypes._minvals
-max_filler.update([(k, -np.inf) for k in [np.float32, np.float64]])
+max_filler.update([(k, -np.inf) for k in float_types_list[:4]])
+max_filler.update([(k, complex(-np.inf, -np.inf)) for k in float_types_list[-3:]])
+
 min_filler = ntypes._maxvals
-min_filler.update([(k, +np.inf) for k in [np.float32, np.float64]])
-if 'float128' in ntypes.typeDict:
-    max_filler.update([(np.float128, -np.inf)])
-    min_filler.update([(np.float128, +np.inf)])
+min_filler.update([(k,  +np.inf) for k in float_types_list[:4]])
+max_filler.update([(k, complex(+np.inf, +np.inf)) for k in float_types_list[-3:]])
```

Comment:
Woops, test failure, probably my typo, mus tbe `min` here...

Reply:
Yup, fixed it

Reply:
@vrakesh I think the push went bad again?!

Reply:
Force pushed again, hopefully doesn't go wrong again. Reconfigured my local git